open-source-license-acknowledgements-and-third-party-copyrights.md

```diff
@@ -0,0 +1,42 @@
+# Open Source License Acknowledgements and Third-Party Copyrights
+
+Operations-Dashboard is licensed under the terms of the [MIT License](./LICENSE).
+
+Operations-Dashboard utilizes third party software from various sources. Portions of this software are copyrighted by their respective owners as indicated in the copyright notices below.
+
+The following acknowledgements pertain to this software license.
+
+### d3
+License: BSD 3-Clause "New" or "Revised" License<br/> 
+https://github.com/d3/d3/blob/master/LICENSE<br/> 
+Copyright 2010-2017 Mike Bostock
+
+### d3-legend d3 plugin
+License: Apache License 2.0<br/> 
+https://github.com/susielu/d3-legend/blob/master/LICENSE<br/> 
+Copyright (c) 2015, Susie Lu
+
+### jQuery
+License: MIT<br/> 
+https://github.com/jquery/jquery/blob/master/LICENSE.txt<br/> 
+Copyright JS Foundation and other contributors, https://js.foundation/
+
+### tablesorter jQuery Plugin
+License: MIT<br/> 
+https://github.com/christianbach/tablesorter/blob/master/LICENSE<br/> 
+Copyright (c) 2014 Christian Bach
+
+### Bootstrap - front-end framework
+License: MIT<br/> 
+https://github.com/twbs/bootstrap/blob/master/LICENSE<br/> 
+Copyright (c) 2011-2019 Twitter, Inc.<br/> 
+Copyright (c) 2011-2019 The Bootstrap Authors
+
+### Font awesome
+License: Font Awesome Free License<br/> 
+https://github.com/FortAwesome/Font-Awesome/blob/master/LICENSE.txt
+
+## Sources of information about licenses
+The GitHub repositories of these packages are used as the source of the information about licenses.
+
```

Comment:
remove unnecessary empty lines

Reply:
I have removed the lines. Also note that headings have spacing around them by default.